tensorboardX/writer.py

```diff
@@ -226,6 +227,10 @@ def __init__(
             filename_suffix: Optional[str] = '',
             write_to_disk: Optional[bool] = True,
             log_dir: Optional[str] = None,
+            comet: Optional[bool] = False,
```

Comment:
Can we combine workspace, project_name, api_key as a dictionary argument (`comet_config`)?  In this way, the arguments are more organized and `comet: Optional[bool]` can be replaced by checking if the dictionary is empty.

Reply:
Sure, that should work fine I think. @ammaddd could you take care of that?

Reply:
I am checking it. But what if the dictionary is empty but the environment variables are set. In that case, the 'dict check' would fail, right?